src/GraphQLProjectConfig.ts

```diff
@@ -40,9 +43,19 @@ export class GraphQLProjectConfig {
   }
 
   includesFile(fileUri: string): boolean {
-    const filePath = fileUri.startsWith('file://')
-      ? fileUri.substr(7)
-      : fileUri;
+    let filePath = fileUri;
+
+    if (fileUri.startsWith('file://')) {
+      if (isWindows) {
+        filePath = filePath.substr(8);
```

Comment:
Why do we use `8` here? What if there's `file://foo.js`

Reply:
It just removes the `file://`, it should work the same